frontend/src/components/globalSearch/GlobalSearchContent.tsx

```diff
@@ -146,17 +146,19 @@ export function GlobalSearchContent({
   // Resource search items
   const resources = useSearchResources();
   const loading = resources.filter(it => it.isLoading).map(it => it.kind);
+  const isMap = history.location.pathname.includes('/map');
```

Comment:
I think this may yield false positives. Maybe you have to compare the current map route using the react-route-dom functions.

Reply:
replaced with useRouteMatch and also got the path from the route definition